src/test/rustdoc/issue-19190.rs

```diff
@@ -23,9 +23,6 @@ impl Deref for Bar {
     fn deref(&self) -> &Foo { loop {} }
 }
 
-// @has issue_19190/Bar.t.html
```

Comment:
You should either include a test that uses the flag or add it to existing tests that check for these pages.

```rust
// compile-flags:--generate-redirect-pages
```